README.md

```diff
@@ -166,7 +166,34 @@ return this.infinityModel("product", { perPage: 12, startingPage: 1,
                                        category: "furniture" });
 ```
 
-Bound Parameters has been DEPRECATED as the third argument to `infinityModel`.  
+**Extending infinityModel**
+
+You can customize your infinityModel through configuring it like so and pass as the third argument to infinityModel:
```

Comment:
As of 1.0+, you can override or extend the behavior of Ember Infinity by providing a class that extends `InfinityModel` as a third argument to the `Route#infinityModel` hook.

**Note:** This behavior should negate any need for the pre 1.0 "Bound Params" style of work.  See [Bound Parameters][Bound Parameters] Section below for more information.